module/debugger_cli/src/mod_debugger_cli.c

```diff
@@ -103,7 +119,9 @@ static int debugger_cli_process_event(const struct fwk_event *event,
     case DEBUGGER_CLI_INTERNAL_EVENT_IDX_ENTER_DEBUGGER:
         /* Start the CLI, blocking the rest of the event queue */
         cli_print("[CLI_DEBUGGER_MODULE] Entering CLI\n");
+        stop_alarm(event->target_id);
```

Comment:
Please make sure you use the returned value from start_ & stop_ functions.
Thanks for the improvements :)

Reply:
I pushed a new version with checks for the return values of the start & stop functions